StylableSwiftUI/Classes/WithStyleIdentifier.swift

```diff
@@ -60,31 +65,65 @@ public extension WithStylistIdentifier {
 
     init(token: String,
                 @ViewBuilder contents: @escaping (StylistIdentifier) -> Content) {
-        self.tokens = [ token ]
+        self.tokens = [ StylistIdentifier(token) ]
         self.contents = { contents($0[0]) }
     }
 
     init(tokens token1: String, _ token2: String,
                 @ViewBuilder contents: @escaping (StylistIdentifier, StylistIdentifier) -> Content) {
-        self.tokens = [ token1, token2 ]
+        self.tokens = [ StylistIdentifier(token1), StylistIdentifier(token2) ]
         self.contents = { contents($0[0], $0[1]) }
     }
 
     init(tokens token1: String, _ token2: String, _ token3: String,
                 @ViewBuilder contents: @escaping (StylistIdentifier, StylistIdentifier, StylistIdentifier) -> Content) {
-        self.tokens = [ token1, token2, token3 ]
+        self.tokens = [ StylistIdentifier(token1), StylistIdentifier(token2), StylistIdentifier(token3) ]
         self.contents = { contents($0[0], $0[1], $0[2]) }
     }
 
     init(tokens token1: String, _ token2: String, _ token3: String, _ token4: String,
                 @ViewBuilder contents: @escaping (StylistIdentifier, StylistIdentifier, StylistIdentifier, StylistIdentifier) -> Content) {
-        self.tokens = [ token1, token2, token3, token4 ]
+        self.tokens = [ StylistIdentifier(token1), StylistIdentifier(token2), StylistIdentifier(token3), StylistIdentifier(token4) ]
```

Comment:
Could these four init methods just call out to the other ones i.e.

```swift
self.init(tokens: tokens.map(StylistIdentifier.init), contents: contents)
```

I can't tell if that complication is worse than the duplication :) Or if it's so small it doesn't matter!

Reply:
oh, no, no then can't - damn variadic arguments!